python/examples/advanced/Profile_Store/LocalStore/generate_data.py

```diff
@@ -0,0 +1,21 @@
+import time
+
+import pandas as pd
+
+import whylogs as why
+from whylogs.api.store.profile_store import ProfileStore
+
+logger = why.logger(mode="rolling", interval=10, when="S")
+logger.append_store(store=ProfileStore(base_name="test_constraints"))
+
+
```

Comment:
i assume that it should only flush based on the config of the rolling logger. 
why would you want to "instantly" flush instead of rolling every X seconds?

Reply:
and users could always use the regular LocalStore api, as in:
```python
store = LocalStore(base_name="test_constraints")
store.write(profile=some_profile)

store.get(config=DateConfig(some_datas_timestamp))
```

Reply:
i'm here assuming `logger.log()` and `store.get()` will happen on different processes. 1) capture 2) compare